Umbraco-Cloud/Upgrades/Migrating-from-8-to-9/index.md

```diff
@@ -38,13 +38,13 @@ Should something fail during the migration, the Development environment can alwa
 
```

Comment:
```suggestion
Create a backup of the database from your Umbraco 8 project, this can be done by using the [database backup guide](https://our.umbraco.com/documentation/umbraco-cloud/Databases/Backups/) or by cloning down the V8 project and making a backup of the local Database. Make sure to restore the content from your cloud environment.
```

Reply:
This should already be there 🤔